js/binance.js

```diff
@@ -907,7 +909,7 @@ module.exports = class binance extends Exchange {
                     'EOS': 'EOS',
                     'SPL': 'SOL',
                 },
-                'reverseNetworks': {
+                'networksById': {
```

Comment:
@ttodua agreed 100%

Reply:
> @ttodua agreed 100%

I will implement unified networks for binance too after huobi is merged, so it will just make `reverseNetworks` obsolete naturally (of course, with temporary optional switch or something, will share my thoughts in separate PR later).